samples/bluetooth/broadcast_config_tool/src/main.c

```diff
@@ -2235,6 +2348,7 @@ SHELL_STATIC_SUBCMD_SET_CREATE(
 		       cmd_frame_interval),
 	SHELL_COND_CMD(CONFIG_SHELL, pd, NULL, "Set presentation delay (us)", cmd_pd),
 	SHELL_COND_CMD(CONFIG_SHELL, file, &sub_file_cmd, "File commands", NULL),
+	SHELL_COND_CMD(CONFIG_SHELL, fixed_id, NULL, "Set fixed broadcast ID", cmd_fixed_id),
```

Comment:
Is the term `fixed_id` something that is used in the spec? Otherwise I think it would be easier to understand this command if it was called `broadcast_id`, as that is what is being set. And then the second argument (`<0/1>`) can still be used to mean fixed/random

Reply:
Have changed to using subcommands after discussion